themes/uhsg_theme/js/accordion.js

```diff
@@ -5,6 +5,12 @@
       $('.accordion__title', context).once().on('click', function () {
         $(this).toggleClass('is-active').next('.accordion__content').toggleClass('visually-hidden');
       });
+      $('.accordion__item', context).once().on('keypress', function (event) {
+        if (event.keyCode != 9) {
```

Comment:
Can we comment what `event.keyCode` `9` means? That's `Tab` I guess?

Reply:
Added comment.